src/test/java/org/zaproxy/zap/extension/hud/ui/browser/tutorial/FramesPageUnitTest.java

```diff
@@ -121,7 +121,7 @@ public void testSidePanelsHiddenAndRevealed(WebDriver driver) throws Exception {
     }
 
     @TestTemplate
-    public void testBottonDrawerTabsHiddenAndRevealed(WebDriver driver) throws URISyntaxException {
+    public void testButtonDrawerTabsHiddenAndRevealed(WebDriver driver) throws URISyntaxException {
```

Comment:
I'm assuming this is the correct intended term

Reply:
Actually I think this shoukd gave been "Bottom"